blockly-core/core/ui/block_svg/block_svg_functional.js

```diff
@@ -51,7 +51,7 @@ Blockly.BlockSvgFunctional.prototype.renderDraw_ = function(iconWidth, inputRows
 
   this.blockClipRect_.setAttribute('d', this.svgPath_.getAttribute('d'));
 
-  var rect = this.svgPath_.getBoundingClientRect();
+  var rect = this.svgPath_.getBBox();
```

Comment:
omg, my local changeset (investigating block jiggling, unfortunately not the cause):

![image](https://cloud.githubusercontent.com/assets/206973/6989984/d8323f1a-da17-11e4-950f-0e077a879277.png)